source/triggers/scheduled-triggers.txt

```diff
@@ -113,11 +116,34 @@ Scheduled Triggers have the following configuration options:
        
        For scheduled Triggers, this value should be set to ``SCHEDULED``.
 
-   * - | :guilabel:`Trigger Name`
+   * - | :guilabel:`Name`
        | ``name: <string>``
 
      - Required. The name of the Trigger.
 
+   * - | :guilabel:`Enabled`
+       | ``disabled: <boolean>``
+
+     - Enabled by default. Used to enable trigger. 
+   
+   * - | :guilabel:`Skip Events on Re-Enable`
+       | ``skip_catchup_event: <boolean>``
+
+     - Disabled by default. When option is enabled, missed trigger events 
+       while trigger is disabled are skipped when trigger is re-enabled. 
```

Comment:
same as above -- why lowercase "t" trigger throughout this description?

Reply:
In favor of changing them to lowercase